### PR TITLE
Add walkthrough to README

### PR DESCRIPTION
**Pull request by nbon12**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -234,6 +234,93 @@ Below is an example usage of ``nise`` for GCP data::
...
+            - cpu_usage:
+                full_period: 1
+            - mem_usage_gig: 
+                full_period: 2
+        
+
+Note: tags in this file are akin to user defined tags in AWS
+Note: If you don't set cost and rate, the generated cost and rate would be random each day.
+
+There example yamls are in the nise repo.
```
Comment (myersCody) [inline README.rst]: I think this needs to be `There are example yamls in the nise repo`

File: README.rst [hunk 2]
```diff
@@ -234,6 +234,93 @@ Below is an example usage of ``nise`` for GCP data::
...
+    - For aws: koku/testing/local_providers/aws_local
+    - For Azure: koku/testing/local_providers/azure_local
+
+=================
+Nise walkthrough
+=================
+This is an example of using nise to generate static report. 
+Run one of the following examples:
+So this creates AWS data from this starting date:
+nise --start-date 2018-06-03
```
Comment (myersCody) [inline README.rst]: This command doesn't work: 
```
(nise) bash-3.2$ nise --start-date 2018-06-03
usage: nise [-h] [--start-date DATE] [--end-date DATE]
            (--aws | --azure | --ocp | --gcp)
            [--aws-s3-bucket-name BUCKET_NAME]
            [--aws-s3-report-name COST_REPORT_NAME]
            [--aws-s3-report-prefix PREFIX_NAME]
            [--aws-finalize FINALIZE_REPORT]
            [--azure-container-name AZURE_CONTAINER_NAME]
            [--azure-report-name AZURE_COST_REPORT_NAME]
            [--azure-report-prefix AZURE_PREFIX_NAME]
            [--static-report-file STATIC_REPORT_FILE]
            [--ocp-cluster-id OCP_CLUSTER_ID]
            [--insights-upload UPLOAD_ENDPOINT]
            [--gcp-report-prefix GCP_REPORT_PREFIX]
            [--gcp-bucket-name GCP_BUCKET_NAME]
            [--azure-account-name AZURE_ACCOUNT_NAME]
nise: error: one of the arguments --aws --azure --ocp --gcp is required
```

File: README.rst [hunk 3]
```diff
@@ -234,6 +234,93 @@ Below is an example usage of ``nise`` for GCP data::
...
+
+this command creates:
+nise --start-date 2018-06-20 --aws --aws-s3-bucket-name testbucket --aws-s3-report-name cpu_cores
+
+how to create OCP data, and check that it's updated for specific provider:
+nise --start-date 2018-06
+The upload path matches where the data is
```
Comment (myersCody) [inline README.rst]: I think we may want to mention that this path is under koku. In fact we may even want to rename this section to using nise with koku or something like that.

**Comment by maskarb** [comment]
#251 cleans up some documentation.

**Pull request closed without merging**